stixcore/ephemeris/manager.py

```diff
@@ -89,29 +133,115 @@ def scet_to_datetime(scet):
             Datetime of SCET
 
         """
+        ephemeris_time = None
         if isinstance(scet, (float, int)):
-            ephemeris_time = spice.sct2e(SOLAR_ORBITER_NAIF_ID, scet)
+            ephemeris_time = spiceypy.sct2e(SOLAR_ORBITER_ID, scet)
         elif isinstance(scet, str):
-            ephemeris_time = spice.scs2e(SOLAR_ORBITER_NAIF_ID, scet)
-        return spice.et2datetime(ephemeris_time)
+            ephemeris_time = spiceypy.scs2e(SOLAR_ORBITER_ID, scet)
+        return spiceypy.et2datetime(ephemeris_time)
 
-    @staticmethod
-    def datetime_to_scet(datetime):
+    @spice_context
+    def datetime_to_scet(self, adatetime):
         """
         Convert datetime to SCET.
 
         Parameters
         ----------
-        datetime : `datetime.datetime` or `astropy.time.Time`
+        adatetime : `datetime.datetime` or `astropy.time.Time`
             Time to convert to SCET
 
         Returns
         -------
         `str`
             SCET of datetime encoded as spacecraft clock string
         """
-        if isinstance(datetime, Time):
-            datetime = datetime.to_datetime()
-        et = spice.datetime2et(datetime)
-        scet = spice.sce2s(SOLAR_ORBITER_NAIF_ID, et)
+        if isinstance(adatetime, ApTime):
+            adatetime = adatetime.to_datetime()
+        et = spiceypy.datetime2et(adatetime)
+        scet = spiceypy.sce2s(SOLAR_ORBITER_ID, et)
         return scet
+
+
+class Position(SpiceManager):
+    """
+    Obtain spacecraft position and orientation, convert to and from instrument coordinate system
+    """
+    spice_context = SpiceManager.spice_context
+
+    @spice_context
+    def get_position(self, *, date, frame):
+        """
+        Get the position of SolarOrbiter.
```

Comment:
Get the position of SolarOrbiter **at a given time and coordinate frame.** 

Is this clear because of the parameters